Library/Homebrew/cmd/audit.rb

```diff
@@ -186,6 +186,7 @@ def audit_file
       [/^  keg_only/,                      "keg_only"],
       [/^  option/,                        "option"],
       [/^  depends_on/,                    "depends_on"],
+      [/^  (go_)?resource/,                "resources"],
```

Comment:
No, you're right, I somehow missed that part of your OC.

I'm fine with it not being pluralised, just thought it sounded a bit weird, but guess that applies to several elements here.